app/(pages)/orders/view-order/_components/Navigator/Navigator.tsx

```diff
@@ -0,0 +1,83 @@
+'use client';
+
+import React from 'react';
```

Comment:
pretty sure we never need to import React in Next.js. Remove this line from all files where you added it. You can do Ctrl/Cmd + Shift + F to do a search in all files if you're using Vscode.